### ISSUE TITLE
Dark Mode renders "Problems" page difficult to read

### ISSUE THREAD

**Issue by dlerner570** [manyfold3d/manyfold#2299]
When the OS or browser is in Dark Mode, the "Problems" page becomes difficult to read.   Each line is rendered with white text on the background color:

<img width="808" alt="Screenshot 2024-06-21 063720" src="https://github.com/manyfold3d/manyfold/assets/165425479/cef8b4f8-e586-4440-b7f6-540e704a7403">

When manually toggled back to light mode, the text is now black on background color:
![Screenshot 2024-06-21 at 06-49-21 Manyfold](https://github.com/manyfold3d/manyfold/assets/165425479/7173df99-729f-45e5-87c3-e7f0203d3aa6)

**Reproduction**

Toggle back and forth between light and dark mode in browser

**Expected behavior**
Text should be readable based on the background color of each line.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Firefox
 - Version 127.0
 - ManyFold:  0.69

**Comment by Floppy** [comment]
@aneurinprice I know you're looking at this; it looks like Boostrap won't have a fix to this until v6 (see https://getbootstrap.com/docs/5.3/content/tables/#variants), so whatever hacky workaround you can come up with will be fine until then :)

**Comment by aneurinprice** [comment]
> so whatever hacky workaround you can come up with will be fine until then :)

Super! I've got a workaround for now. Will get it pushed later today.

**Comment by Floppy** [comment]
Looks good @aneurinprice !

**Comment by beniroquai** [comment]
Hey, sorry, I know this is outdataed and closed, but was there a possibility to change from bright to darkmode with a switch inside manyfold somewhere (so not by changing it on the Browser/OS level). Thanks :)

**Comment by Floppy** [comment]
I definitely want to add a manual option, but Bootstrap (the CSS framework we use) can either be compiled in manual or automatic mode, not both. I'm hoping that will change in version 6 and we can provide both options.
